dff/stats/cli.py

```diff
@@ -0,0 +1,251 @@
+
+:meta hide-value:
+"""
+
+DFF_NODE_STATS_STATEMENT = """
+WITH main AS (
+    SELECT DISTINCT {table}.LogAttributes['context_id'] as context_id,
+    {table}.LogAttributes['request_id'] as request_id,
+    toDateTime(otel_traces.Timestamp) as start_time,
+    otel_traces.SpanName as data_key,
+    {table}.Body as data,
+    {lblfield} as label,
+    {flowfield} as flow_label,
+    {nodefield} as node_label,
+    {table}.TraceId as trace_id,
+    otel_traces.TraceId\nFROM {table}, otel_traces
+    WHERE {table}.TraceId = otel_traces.TraceId and otel_traces.SpanName = 'get_current_label'
+    ORDER BY context_id, request_id
+) SELECT context_id,
+    request_id,
+    start_time,
+    data_key,
+    data,
+    label,
+    {lag} as prev_label,
+    flow_label,
+    node_label
+FROM main
+WHERE label != ''
+"""
+DFF_ACYCLIC_NODES_STATEMENT = """
+WITH main AS (
+    SELECT DISTINCT {table}.LogAttributes['context_id'] as context_id,
+    {table}.LogAttributes['request_id'] as request_id,
+    {table}.Timestamp as timestamp,
+    {lblfield} as label\nFROM {table}
+    INNER JOIN
+(
+    WITH helper AS (
+        SELECT DISTINCT {table}.LogAttributes['context_id'] as context_id,
+        {table}.LogAttributes['request_id'] as request_id,
+        {lblfield} as label
+        FROM {table}
+    )
+    SELECT context_id FROM helper
+    GROUP BY context_id
+    HAVING COUNT(context_id) = COUNT(DISTINCT label)
+) as plain_ctx
+ON plain_ctx.context_id = context_id
+ORDER by context_id, request_id
+)
+SELECT * FROM main
+"""
+DFF_FINAL_NODES_STATEMENT = """
+WITH main AS (
+    SELECT LogAttributes['context_id'] AS context_id,
+    max(LogAttributes['request_id']) AS max_history
+    FROM {table}\nGROUP BY context_id
+)
+SELECT DISTINCT LogAttributes['context_id'] AS context_id,
+LogAttributes['request_id'] AS request_id,
+{table}.Timestamp AS start_time,
+{lblfield} AS label,
+{flowfield} AS flow_label,
+{nodefield} AS node_label
+FROM {table}
+INNER JOIN main
+ON context_id  = main.context_id
+AND request_id = main.max_history
+INNER JOIN otel_traces
+ON {table}.TraceId = otel_traces.TraceId
+WHERE otel_traces.SpanName = 'get_current_label'
+"""
+
+SQL_STATEMENT_MAPPING = {
+    "dff_acyclic_nodes.yaml": DFF_ACYCLIC_NODES_STATEMENT,
+    "dff_node_stats.yaml": DFF_NODE_STATS_STATEMENT,
+    "dff_final_nodes.yaml": DFF_FINAL_NODES_STATEMENT,
+}
+"""
+Select statements for dashboard configuration with names and types represented as placeholders.
+The placeholder system makes queries database agnostic, required values are set during the import phase.
+
+:meta hide-value:
+"""
+
+
+def import_dashboard(parsed_args: Optional[argparse.Namespace] = None, zip_file: Optional[str] = None):
+    """
+    Import an Apache Superset dashboard to a local instance with specified arguments.
+    Before using the command, make sure you have your Superset instance
+    up and running: `ghcr.io/deeppavlov/superset_df_dashboard:latest`.
```

Comment:
Mention here and in the help message of `dff.stats` that running it overwrites previous dashboard configuration.